tests/main/cgroup-devices/task.yaml

```diff
@@ -0,0 +1,8 @@
+summary: measuring basic properties of device cgroup
+# Ubuntu 14.04 uses a custom build of an older systemd that does not
+# seem to use device cgroups for services. The test is invalid there.
+systems: [-ubuntu-14.04-*]
+execute: ./task.sh
```

Comment:
Why there is a .sh file instead of adding the body in the execute part of the test?

Reply:
Because it is equivalent except my editor knows what I'm doing then.